mllib/src/main/scala/org/apache/spark/ml/r/GaussianMixtureWrapper.scala

```diff
@@ -124,7 +129,8 @@ private[r] object GaussianMixtureWrapper extends MLReadable[GaussianMixtureWrapp
       val rMetadataStr = sc.textFile(rMetadataPath, 1).first()
       val rMetadata = parse(rMetadataStr)
       val dim = (rMetadata \ "dim").extract[Int]
-      new GaussianMixtureWrapper(pipeline, dim, isLoaded = true)
+      val logLikelihood = (rMetadata \ "logLikelihood").extract[Double]
+      new GaussianMixtureWrapper(pipeline, dim, logLikelihood, isLoaded = true)
```

Comment:
it may not be a big deal right now, since spark.gmm is relatively new.
but I think we should come up with a plan on model persistent compability not only with R vs JVM but also across versions of Spark.
also might be useful to link this JIRA to SPARK-18864